src/main/app/controller/PostController.ts

```diff
@@ -78,6 +78,29 @@ export class PostController<T extends AnyObject> {
     });
   }
 
+  public async sendToApplicant2ForReview(req: AppRequest<T>, res: Response, formData: Partial<Case>): Promise<void> {
```

Comment:
Would it still worked if you moved this method out of `PostController.ts` and into `InviteApplicant2PostController.ts`?

Reply:
👍

Reply:
After trying it out unfortunately the suggestion didn't work due to this method using the private `save` method in the PostController.

Reply:
I think I agree with Seb, this is behaviour specific to a page and shouldn't be in the general controller. If you want to re-use the save method you could subclass and make it protected but it's possible you don't need that behaviour and just want to trigger the event